README.md

```diff
@@ -32,9 +32,11 @@ use Navarr\Socket\Server;
 
 class EchoServer extends Server
 {
-    public function __construct($ip = null, $port = 7)
+    const DEFAULT_PORT = 7;
+    
+    public function __construct($ip = null, $port = DEFAULT_PORT)
```

Comment:
`DEFAULT_PORT` in the constructor does not appear to be properly scoped.  I believe in this case it should be `static::DEFAULT_PORT`.